tests/common.go

```diff
@@ -215,6 +230,30 @@ func StopVolDriverAndWait(appNodes []node.Node) {
 	})
 }
 
+// CrashVolDriverAndWait crashes volume driver on given app nodes and waits till driver is back up
+func CrashVolDriverAndWait(appNodes []node.Node) {
+	context(fmt.Sprintf("crashing volume driver %s", Inst().V.String()), func() {
+		Step(fmt.Sprintf("crash volume driver on nodes: %v", appNodes), func() {
+			for _, n := range appNodes {
+				err := Inst().V.StopDriver(n, true)
+				expect(err).NotTo(haveOccurred())
+			}
+		})
+
+		Step(fmt.Sprintf("Sleeping for %v for crash to take effect", waitVolDriverToCrash), func() {
```

Comment:
isn't this a lot of time to wait unconditionally? I feel the crash and wait time should be inside the storage driver implementation and not in the test.
and it would vary based on the CO as well